README.md

```diff
@@ -30,12 +30,6 @@ A transaction is displayed.
 
 * Install node v20
 
-### To Install
```

Comment:
I thought it was confusing since `npm install` is to install dependencies, not install the wallet.
And I think we can leave dependency installation out of the readme unless we have some extra steps to install dependencies other than the default `npm install`.

What do you think?

Reply:
This section is a helper to run the wallet for development, so I think it makes sense to have a sub section to install dependencies. We could change the name of the sub section to "Install dependencies" if you prefer. What do you think?